solr/cross-dc-manager/src/java/org/apache/solr/crossdc/manager/consumer/Consumer.java

```diff
@@ -21,8 +21,8 @@
 import static org.apache.solr.crossdc.common.KafkaCrossDcConf.ZK_CONNECT_STRING;
 
 import com.codahale.metrics.SharedMetricRegistries;
-import com.codahale.metrics.servlets.MetricsServlet;
-import com.codahale.metrics.servlets.ThreadDumpServlet;
+import io.dropwizard.metrics.servlets.MetricsServlet;
```

Comment:
it's suspicious to see we import com.codahale & io.dropwizard.  Maybe okay but I wasn't expecting.